bigframes/ml/pipeline.py

```diff
@@ -101,8 +102,8 @@ def _from_bq(cls, session: bigframes.Session, bq_model: bigquery.Model) -> Pipel
 
     def fit(
         self,
-        X: Union[bpd.DataFrame, bpd.Series],
-        y: Optional[Union[bpd.DataFrame, bpd.Series]] = None,
+        X: Union[bpd.DataFrame, bpd.Series, pd.DataFrame, pd.Series],
+        y: Optional[Union[bpd.DataFrame, bpd.Series, pd.DataFrame, pd.Series]] = None,
```

Comment:
Do you think we can define a single annotation representing this set of types and use it everywhere. This will make it easier to accomodate additional types in the future

Reply:
My main concern is that type aliases may not be expanded/resolved when generating docs, and thus confuse our end users.

Reply:
It seems that sklearn doesn't try to define a set of types in its docs: https://scikit-learn.org/1.5/modules/generated/sklearn.linear_model.LinearRegression.html#sklearn.linear_model.LinearRegression.fit ?

Reply:
Updated all to use type alias because Sphinx is able to resolve them